### DISCUSSION TITLE
possible to have hand tool to move the view?

### DISCUSSION THREAD

**Discussion by leyu-gopuff** [excalidraw/excalidraw#4156] [Q&A]
Hi, if it has not been discussed before. Please let me know the conclusion if it has been settled. 

Looks like for now we have to zoom out and put the mouse pointer in the view and zoom in, via this way to navigate through the diagram. Would it be possible to have a hand tool other than the selection? Thanks!

**Comment by dwelle** [comment]
Hello.

On the roadmap (https://github.com/excalidraw/excalidraw/issues/794). In the meantime, you can use mouse middle button, or drag while holding space.

Refer to the help:

![image](https://user-images.githubusercontent.com/5153846/139653672-a9437595-7cfd-413a-a174-3b83bc3d0b9b.png)
